pkg/controller/strategy/comparisons.go

```diff
@@ -1,11 +1,23 @@
 package strategy
 
-func contenderCapacityComparison(achieved uint, desired uint) bool {
-	return achieved >= desired
+func contenderCapacityComparison(achievedPercentage, desiredPercentage, totalNumberOfPods uint) bool {
+	// Consider that contender capacity has achieved its desired state if
```

Comment:
I think it might make sense to explain here and in the `incumbentCapacityComparison` about our "pod math", where we always round up to the nearest whole pod (and so 10% desired capacity of 1 pod is 1 pod).